src/ucs/type/wakeup_event.h

```diff
@@ -0,0 +1,18 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2016.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_TYPE_WAKEUP_EVENT_H
+#define UCS_TYPE_WAKEUP_EVENT_H
+
+#include <ucs/sys/math.h>
+
+typedef enum ucs_wakeup_event_types {
```

Comment:
this should not be in UCS because the concept of "wakeup" is specific for communications - this is not a generic infrastructure